kubernetes/docs/includes/deploy-jars-k8s.rst

```diff
@@ -1,36 +1,23 @@
-Deploying Custom Kafka Connect JARs with Operator 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Deploying Connectors with Operator 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-`Kafka Connect <https://docs.confluent.io/current/connect/index.html>`__ employs an extensible programming model to faciliate `Data Conversions <https://docs.confluent.io/current/connect/concepts.html#converters>`__, `Single Message Transformations (SMTs) <https://docs.confluent.io/current/connect/concepts.html#transforms>`__, or the development of custom connectors if a `supported connector <https://www.confluent.io/hub/>`__ is not available.  The Connect `packaging model <https://docs.confluent.io/current/connect/devguide.html#packaging>`__ will require that you deploy an archive containing your custom JAR to the `Kafka Connect's plugin path <https://docs.confluent.io/current/connect/userguide.html#connect-installing-plugins>`__.
+|c-hub| is an online library of pre-packaged and ready-to-install connectors, transformations, and converters that suit your needs.
```

Comment:
what kinda stinks about the doc includes is there is no link... i think a link to CHub would be helpful here

Reply:
@rspurgeon IIUC, you are pointing out that this links to a docs page that doesn't have a link to CHub, right?  If so, please file a DOCS jira to get it fixed ;)

Reply:
Not exactly, I meant that by using the doc replacement for |c-hub| it just puts in the text, but there is no direct link from here to https://www.confluent.io/hub/

It's a nit, but might save someone a google search so seemed helfpul

Reply:
Ack and fixed!